src/main/resources/avro/bdg.avdl

```diff
@@ -883,57 +884,141 @@ record TranscriptEffect {
 }
 
 /**
- Annotations of a variant, as produced by a tool such as SnpEff or Ensembl VEP.
+ Variant annotation.
  */
 record VariantAnnotation {
 
   /**
    Variant for this annotation.
    */
-  union { null, Variant } variant;
+  union { null, Variant } variant = null;
 
   /**
-   Zero or more transcript effects, per alternate allele and transcript (or other feature).
+   Ancestral allele, VCF INFO reserved key AA, Number=1, shared across all alleles
+   in the same VCF record.
    */
-  array<TranscriptEffect> transcriptEffects = [];
-}
+  union { null, string } ancestralAllele = null;
 
-/**
- Database variant annotations.
- */
-record DatabaseVariantAnnotation {
-  union { null, Variant } variant;
+  /**
+   Allele count, VCF INFO reserved key AC, Number=A, split for multi-allelic sites.
+   */
+  union { null, int } alleleCount = null;
+
+  /**
+   Total read depth, VCF INFO reserved key AD, Number=R, split for multi-allelic
+   sites.
+   */
+  union { null, int } readDepth = null;
+
+  /**
+   Forward strand read depth, VCF INFO reserved key ADF, Number=R, split for
+   multi-allelic sites.
+   */
+  union { null, int } forwardReadDepth = null;
+
+  /**
+   Reverse strand read depth, VCF INFO reserved key ADR, Number=R, split for
+   multi-allelic sites.
+   */
+  union { null, int } reverseReadDepth = null;
+
+  /**
+   Minor allele frequency, VCF INFO reserved key AF, Number=A, split for multi-allelic
+   sites. Use this when frequencies are estimated from primary data, not calculated
+   from called genotypes.
+   */
+  union { null, float } alleleFrequency = null;
```

Comment:
The doc above is from the VCF spec, but we write to this field (well, its predecessor) in Quinine, see
https://github.com/bigdatagenomics/quinine/blob/8576f970e43de33797de36f6f622a6b5ef592736/quinine-core/src/main/scala/org/bdgenomics/quinine/annotations/AlleleFrequency.scala#L66